### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by bovirus**
Please take care about strange replacement in source strings of space as mentioned in zh_tw new translation.

### CODE REVIEW

File: recovery/translation_it.ts [hunk 1]
```diff
@@ -101,22 +101,22 @@
...
         <source>&amp;Up</source>
         <oldsource>Up</oldsource>
-        <translation type="unfinished">Su</translation>
+        <translation>&amp;Su</translation>
     </message>
     <message>
         <location filename="bootselectiondialog.ui" line="52"/>
         <source>U</source>
-        <translation type="unfinished"></translation>
+        <translation>U</translation>
```
Comment (lurch) [inline recovery/translation_it.ts:109]: If you're leaving the shortcut-key as U, should the translation of `&amp;Up` be `S&amp;u` ? (IIRC the `&amp;` causes the next character to be underlined)

**Comment by bovirus** [comment]
OK. Thanks for the clarification.
I was not sure about it.

The translation for "U" is "S" (=Su) and for "D" is "G" (=Giù)

**Comment by procount** [comment]
I have merged the update with a few fixes as detailed above.
Please can you check it.

**Comment by procount** [comment]
Sorry, I've messed up this PR by force pushing. Please just check the latest on Master branch.

**Pull request closed without merging**